web/themes/contrib/civictheme/templates/block/block--system-branding-block.html.twig

```diff
@@ -4,7 +4,7 @@
  * CivicTheme implementation to display a Branding block.
  */
 #}
-{% extends "block.html.twig" %}
+{% extends 'block.html.twig' %}
```

Comment:
Before this PR, CT had a mix of `'` and `"` throughout the codebase. This PR fixes this.